models/agol.js

```diff
@@ -990,6 +990,7 @@ var AGOL = function (koop) {
           if (err) agol.log.error(err)
           try {
             error = JSON.parse(value)
+            if (error.type === 'db') return removeJob(job)
```

Comment:
That happens here: https://github.com/koopjs/koop-pgcache/blob/master/index.js#L581